java-example.json

```
@@ -189,6 +189,44 @@
 					}
 				}
 			}
+		},
+		"Java Swing Example - MultiWindow": {
+			"window": {
+				"id": "MultiWindowJavaSwingExample",
+				"windowType": "native",
+				"path": "$javaExampleJarRoot/MultiWindowJavaSwingExample.jar",
+				"url": "",
+				"arguments": "",
+				"defaultHeight": 600,
+				"autoShow": true,
+				"alwaysOnTop": false,
+				"resizable": true,
+				"showTaskbarIcon": false,
+				"contextMenu": true,
+				"addToWorkspace": true
+			},
+			"component": {
+				"spawnOnStartup": false
+			},
+			"foreign": {
+				"services": {
+					"workspaceService": {
+						"isArrangable": true
```

`isArrangable` is depreciated. 
```suggestion
						"allowAutoArrange": true
```